src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -36,6 +37,7 @@ namespace Microsoft.Health.Fhir.Api.Controllers
     [ServiceFilter(typeof(OperationOutcomeExceptionFilterAttribute))]
     public class ExportController : Controller
     {
+        private static readonly List<string> SupportedOutputFormats = new List<string>(new string[] { "application/fhir+ndjson", "application/ndjson", "ndjson" });
```

Comment:
nit: maybe HashSet instead of List? Also, should we explicitly call out the `IEqualityComparer` to determine how the items should be compared?